test/unit-modcc/test_parser.cpp

```diff
@@ -323,7 +323,6 @@ TEST(Parser, parse_parenthesis_expression) {
         "(x             ",
         "((x+3)         ",
         "(x+ +)         ",
-        "(x=3)          ",  // assignment inside parenthesis isn't allowed
         "(a + (b*2^(x)) ",  // missing closing parenthesis
```

Comment:
If we have `tok::eq` as the default stop token in `parse_expression`, we could keep this test case…

Reply:
Will keep the test.